gdx/src/com/badlogic/gdx/utils/ReflectionPool.java

```diff
@@ -22,7 +22,12 @@
 
 /** Pool that creates new instances of a type using reflection. The type must have a zero argument constructor.
  * {@link Constructor#setAccessible(boolean)} will be used if the class and/or constructor is not visible.
+ * <p>
+ *
+ * The class is deprecated, please use {@link DefaultPool} instead.
+ *
  * @author Nathan Sweet */
+@Deprecated
```

Comment:
I'm not really convinced that ReflectionPool is actually obsolete or that it should be deleted in the future. The existing code that uses ReflectionPool is shorter and more convenient, even if its use of reflection internally causes problems sometimes.

Reply:
> that it should be deleted in the future.

I would keep it indefinitly, I just think discouraging it's use is a good idea, if the DefaultPools acts as a good replacement.